.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ '**' ]
+  pull_request:
+    branches: [ '**' ]
+jobs:
+  test:
+    timeout-minutes: 60
```

Comment:
60 minutes seems pretty long, any reason this can't be like 5?

Reply:
We can change this. Most of this is the default provided by running the command.